doc/admin-guide/plugins/lua.en.rst

```diff
@@ -1903,6 +1903,26 @@ ts.server_request.set_url_scheme
 
 :ref:`TOP <admin-plugins-ts-lua>`
 
+ts.server_request.get_method
+----------------------------
+**syntax:** *ts.server_request.get_method()*
+
+**context:** do_remap/do_os_response or do_global_* or later
```

Comment:
this should only be allowed in TS_LUA_HOOK_SEND_REQUEST_HDR hook point

Reply:
pls check this out as example - https://docs.trafficserver.apache.org/en/latest/admin-guide/plugins/lua.en.html#ts-server-request-get-uri